src/core/backbone_router/multicast_listeners_table.cpp

```diff
@@ -33,7 +33,7 @@
 
 #include "multicast_listeners_table.hpp"
 
-#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
```

Comment:
update the ending comments accordingly

`#endif // OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE	`

Also in the other files